### DISCUSSION TITLE
Searching for tasks - API and UI

### DISCUSSION THREAD

**Discussion by subkanthi** [apache/airflow#18015] [General]
Currently is there a way to search for task names, the UI seems to be only able to search for DAG names.

**Comment by josh-fell** [comment]
Is searching for Task Instances what you are looking for or something else specifically?

You can search for Task Instances via Browse > Task Instances and search by "Task Id":
![image](https://user-images.githubusercontent.com/48934154/132023244-73d57b0c-3370-4cb1-9d91-6e087437ed3f.png)

**Comment by subkanthi** [comment]
I was really hoping in the DAG page, where someone needs to search by task name.
<img width="1416" alt="Screen Shot 2021-09-03 at 10 44 36 AM" src="https://user-images.githubusercontent.com/17434846/132024322-53dd24e1-1751-4258-a9c2-93138fdd58bf.png">
